notebook/static/tree/js/notebooklist.js

```diff
@@ -425,7 +425,7 @@ define([
             }
         );
     };
-    
+
```

Comment:
Should I reinstate this trailing white-space? I got automatically removed by my editor and I think that is what you want but happy to revert it.